contributing.md

```diff
@@ -0,0 +1,37 @@
+# What this repository is
+This repository contains sources of the Introduction to Python course.
+
+# Contributing
+We love contributions!
+We are happy to see fixes of the existing bugs and tasks and tests mistakes.
+The current tasks can be found in the [open issues](https://github.com/jetbrains-academy/introduction_to_python/issues) in the project.
```

Comment:
The current tasks can be found in the [open issues](https://github.com/jetbrains-academy/introduction_to_python/issues) section of the project.